### PR TITLE
Post

### PR DESCRIPTION
**Pull request by saketkum11**
- Home page
- created postSlice
- Edit ,Delete,GetAllPost

### CODE REVIEW

**Review by shubhamkumar648 — APPROVED** [review]
- Edit ,post ,delete feataures  works perfectly 
- Well  written code 
- Ready to merge

File: src/Page/Profile/Profile.jsx [hunk 1]
```diff
@@ -1,10 +1,84 @@
...
+  const dispatch = useDispatch();
+  const { username } = useParams();
+  const [profileEditFlag, setProfileEditFlag] = useState(false);
+  const { individualUser } = useSelector((store) => store.user);
+  const { posts, userPost } = useSelector((store) => store.post);
+
+  const { avatarURL, bio, firstName, website, lastName, followers, following } =
+    individualUser;
+
+  console.log("from profile", posts);
```
Comment (Pratik1005) [inline src/Page/Profile/Profile.jsx]: remove logs if not needed

File: src/backend/controllers/PostController.js [hunk 2]
```diff
@@ -78,6 +78,7 @@ export const createPostHandler = function (schema, request) {
       );
     }
     const { postData } = JSON.parse(request.requestBody);
+    console.log("post data from backend", postData);
```
Comment (Pratik1005) [inline src/backend/controllers/PostController.js]: same here

File: src/backend/controllers/PostController.js [hunk 3]
```diff
@@ -267,6 +268,7 @@ export const deletePostHandler = function (schema, request) {
       );
     }
     const postId = request.params.postId;
+    console.log("post id from postcontroller", postId);
```
Comment (Pratik1005) [inline src/backend/controllers/PostController.js]: same here

**Review by Pratik1005 — APPROVED** [review]
Functionality working properly, good to merge

File: src/backend/controllers/PostController.js (hunk 3 above)
Comment (saketkum11) [reply src/backend/controllers/PostController.js]: done

File: src/backend/controllers/PostController.js (hunk 2 above)
Comment (saketkum11) [reply src/backend/controllers/PostController.js]: done

File: src/Page/Profile/Profile.jsx (hunk 1 above)
Comment (saketkum11) [reply src/Page/Profile/Profile.jsx]: done

File: src/component/PostModal.jsx [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+                  <div className="flex justify-between ">
+                    <div className="flex">
+                      <button className="mr-1">
+                        <i class="fa-solid fa-image"></i>
+                      </button>
+                      <button className="mr-1">
+                        <i class="fa-solid fa-video"></i>
+                      </button>
+                      <button className="mr-1">
+                        <i class="fa-solid fa-face-smile"></i>
```
Comment (sharathnair9999) [inline src/component/PostModal.jsx]: use className here

**Pull request merged**